app/javascript/packs/pw-strength.js

```diff
@@ -85,16 +85,29 @@ export function getFeedback(z) {
   return `${suggestions.map((s) => lookup(s)).join('. ')}`;
 }
 
-function disableSubmit(submitEl, length = 0, score = 0) {
-  if (!submitEl) {
-    return;
-  }
+// function disableSubmit(submitEl, length = 0, score = 0) {
```

Comment:
The function `disableSubmit` is no longer relevant so it has been commented out for now. I think it would be nice to leave the function in the codebase should the feature be re-enabled. The comments will be removed.

Reply:
I think we should remove it if it's not necessary anymore. We can always reference the git history if we need to recall it.